sdks/java/build-tools/src/main/resources/beam/findbugs-filter.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <FindBugsFilter>
   <Bug category="I18N"/>
+  <Bug pattern="DM_STRING_CTOR"/>
```

Comment:
I was also concerned about not seeing others complaining about it. Not really the big value add of findbugs anyhow.